### ISSUE TITLE
Status output one reading behind green energy as of v1.1.6

### ISSUE THREAD

**Issue by ngardiner** [ngardiner/TWCManager#13]
Most likely due to the output being printed within TWCManager, where the green energy check is then queued to be performed in the update thread. May need to move the status print function to the update thread.

```
11:07:14: Solar generating 1074W, Consumption 845W, Charger Load 0W
          Limiting car charging to 4.47A - 3.52A = 2.28A.
          Charge when above 6A (minAmpsPerTWC).

11:08:15: Solar generating 1074W, Consumption 845W, Charger Load 0W
          Limiting car charging to 4.47A - 3.52A = 0.95A.
          Charge when above 6A (minAmpsPerTWC).
```

The first value printed lags the new green energy measurement, hence the incorrect total. The 3rd value always appears to be one measurement behind after a green energy update.

**Comment by ngardiner** [comment]
Nil impact to functionality, this is simply a race condition when printing status update.

**Comment by ngardiner** [comment]
Fixed in 03e3487
